dal/db.go

```diff
@@ -11,40 +11,36 @@ import (
 
 // IkaSignRequest represents a row in the `ika_sign_requests` table.
 type IkaSignRequest struct {
-	// ID is a sign request reported and managed by Native
-	ID        uint64    `json:"id"`
-	Payload   Payload   `json:"payload"`
 	DWalletID string    `json:"dwallet_id"`
 	UserSig   string    `json:"user_sig"`
+	Payload   Payload   `json:"payload"`
 	FinalSig  Signature `json:"final_sig"`
+	ID        uint64    `json:"id"` // ID is a sign request reported and managed by Native
 	Timestamp int64     `json:"time"`
```

Comment:
let's put both Timestamp and ID to the top

Reply:
this will not work. Same as all the below